scripts/delayed.js

```diff
@@ -5,6 +5,9 @@ import { loadScript, sampleRUM } from './aem.js';
 sampleRUM('cwv');
 // add more delayed functionality here
 async function loadAdobeDTM() {
+  if (new URLSearchParams(window.location.search).get('disableLaunch') === 'true') {
```

Comment:
should we check that the window.location.domain to allow this only on `localhost`, but not on .page, .live and production?

Reply:
I would argue it's good to have that on any environment.
Experience is you often run into production issues where you need to isolate whether the issue is injected by launch (or Target or other martech), so a killswitch for launch would be cool to have in such scenarios

Reply:
I'm a bit concerned about people having the ability to disable things like gdpr consent, analytics, etc. in production... maybe I am overly cautious, but at this stage I personally argue to allow this option for localhost and `.page` maximum, as a compromise solution. if we ever need to troubleshoot something in production, we can quickly open a PR, modify the if statement, merge and revert after troubleshooting.